docs/localization.md

```diff
@@ -0,0 +1,165 @@
+# Extension Localization
+
+Creating Extensions is an important feature required to further extend Cognigy AI with additional functionality. As the UI supports several languages in the menus we also added a functionality to add translations to Extensions.
+
+Localization is a completely optional feature that can be used whenever required. You don't need to enforce it consistently throughout an Extension, you can use it as well to translate just a single string.
+
+## How does Localization work?
+
+Localization supports the replacement of simple strings with a JSON object like this:
+
+```json
+{
+	"default": "Default fallback string",
+	"enUS": "String with English Localization",
+	"deDE": "String with German Localization",
+	"esES": "String with Spanish Localization",
+	"koKR": "String with Korean Localization",
+	"jaJP": "String with Japanese Localization"
+}
+```
+
+where the "default" field is mandatory and the other ones are optional, so you can e.g. choose to have an English and a German Localization only
+
+## Where is Localization visible?
+
+Localization will be visible for every user of the Cognigy AI GUI, therefore a Node with a German Localization will be displayed in German for every user with a GUI language of German.
+
+Endusers won't see a difference when communicating with the localized Node
+
+## What can be localized?
+
+Localization doesn't work for all fields. Right now these fields are supported in a Descriptor:
```

Comment:
you keep using field, but I think you mean property? please have another look

Reply:
Thx, changed it